src/test/java/com/jcraft/jsch/JSchTest.java

```diff
@@ -3,10 +3,12 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertSame;
 
-import java.util.Hashtable;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
I'm fairly certain this doesn't meet Google Java style that we utilize.
You should run `mvn formatter:format`.